src/ui/public/courier/saved_object/saved_object_loader.js

```diff
@@ -78,11 +78,12 @@ export class SavedObjectLoader {
    */
   find(searchString, size = 100) {
     let body;
+
     if (searchString) {
       body = {
         query: {
-          simple_query_string: {
-            query: searchString + '*',
+          query_string: {
+            query: /^[\w]*$/.test(searchString) ? `${searchString}*` : searchString,
```

Comment:
I think you probably want `+` instead of `*` in the regex so at least one word character is matched. Also I _think_ you can get rid of the `[` and `]` since you are not specifying multiple characters or character classes.